src/installation/guides/fde.md

```diff
@@ -41,12 +41,16 @@ Device      Start       End   Sectors  Size Type
 /dev/sda2  264192 100663262 100399071 47.9G Linux filesystem
 ```
 
-Configure the encrypted volume. `cryptsetup` defaults to LUKS2, yet GRUB
-currently only has support for LUKS1, so it is critical to force LUKS1. Keep in
-mind this will be `/dev/sda2` on EFI systems.
+`cryptsetup` defaults to LUKS2, yet GRUB currently only has support for LUKS1,
+so it is critical to force LUKS1.
+
+## Encrypted volume configuration
+
+Keep in mind the encrypted volume will be `/dev/sda2` on EFI systems, as the EFI
+partition is `/dev/sda1`.
 
 ```
-# cryptsetup luksFormat --type luks1 /dev/sda1
+# cryptsetup luksFormat --type luks1 /dev/sda2
```

Comment:
This should use `sda1`. We give a warning about it being `sda2` in the paragraph right above this, people should read these in order. Someone doing a BIOS setup can be confused.